package.json

```diff
@@ -8,6 +8,8 @@
   "scripts": {
     "dev": "parcel index.html",
     "dev:proxy": "node api-proxy.js",
+    "type-check": "tsc --noEmit",
+    "prebuild": "npm run type-check",
```

Comment:
curious why we need this, doesn't `parcel` fail to build the app is there are typescript errors?

Reply:
I expected it does, instead `parcel` silently builds the app despite typescript errors.